### PR TITLE
Added ability to change timestamps location

### PR DESCRIPTION
**Pull request by ghost**
This is a follow up for #301

### CODE REVIEW

File: messages_text.go [hunk 1]
```diff
@@ -83,13 +83,17 @@ func (mt *MessagesText) createMessage(m discord.Message) {
...
 		fmt.Fprintf(mt, "[::d]%s", config.Current.Theme.MessagesText.ReplyIndicator)
 	}
 
-	fmt.Fprintf(w, "[%s]%s[-] ", config.Current.Theme.MessagesText.AuthorColor, m.Author.Username)
+	fmt.Fprintf(w, "[%s]%s[-:-:-] ", config.Current.Theme.MessagesText.AuthorColor, m.Author.Username)
 
-	if config.Current.Timestamps {
+	if config.Current.Timestamps && !config.Current.TimestampsBeforeAuthor {
 		fmt.Fprintf(w, "[::d]%s[::-] ", m.Timestamp.Format(time.Kitchen))
 	}
```
Comment (ayn2op) [inline messages_text.go]: Compute the formatted timestamp if the `timestamps` field is set as `true` in the configuration file and then write the computed timestamp to the `io.Writer` depending on the value of `timestamps_before_author` instead.

Comment (ghost) [reply messages_text.go]: So you mean `m.Timestamp.Format(time.Kitchen)` be used once or what?

Comment (ayn2op) [reply messages_text.go]: Yes.

**Comment by ghost** [comment]
OK. Got it to the new project format.

BTW why you moved to kong for CLI?

**Pull request merged**